cmd/monaco/integrationtest/account/deploy-download_test.go

```diff
@@ -94,6 +93,8 @@ func TestIdempotenceOfDeployment(t *testing.T) {
 		assert.Equal(t, deploy1st.Groups[g.ID], deploy2nd.Groups[g.ID])
 	}
 
-	err := monaco("account delete --manifest manifest.yaml --file delete.yaml").withFs(baseFs).run()
-	require.NoError(t, err)
+	t.Cleanup(func() {
+		_, err := test.Monacof("monaco account delete --manifest manifest.yaml --file delete.yaml").WithFs(baseFs).Run()
```

Comment:
nit: I find it confusing that you're sometimes using this prefixed with 'monaco' and other times without. The fact that the prefix is trimmed anyway is hidden deep in this helper method. I'd either consistently use it with 'monaco' or simply drop the trimming and make it clear from the signature/doc that it takes the command, args and flags that are input to the CLI